File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.py

def solution(babbling):
    word = ["aya", "ye", "woo", "ma"]
    answer = 0
    for i in range(len(babbling)):
        for j in range(len(word)):
            if word[j] + word[j] in babbling[i]:
                continue
            if word[j] in babbling[i]:
                babbling [i] = babbling[i].replace(word[j], '*')
                answer = babbling.count('*') + babbling.count('*'*2) + babbling.count('*'*3) + babbling.count('*'*4)
    return answer

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
    let answer = 0;
    const possible = ["aya", "ye", "woo", "ma"]
    babbling.map((word) => {
        possible.forEach((babble) => {
            word = word.replaceAll(babble, "k")
        })
        if(word.replaceAll("k", "") === "") answer++
        return word
    })
    return answer;
}